package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "start": "node index",
     "build": "grunt",
+    "dev": "concurrent 'grunt dev' 'npm start'",
```

Comment:
Beautiful.

For other people, this means we can use browserify without having to run a second command. So you just run "npm run dev" isntead of "npm start" when developing, and watchify will build the build again every time you change any file in the client. I use watchify on a couple of projects, and it's seriously so quick I don't even notice the build script.